config/cron.hjson

```diff
@@ -9,14 +9,14 @@
         with
         enroll_data as (select id as enroll_id, user_id, type from enrollment_dim where course_id='{course_id}'
                         and type in ('StudentEnrollment', 'TaEnrollment', 'TeacherEnrollment') and workflow_state= 'active'),
-        user_info as (select p.unique_name,p.sis_user_id, u.name, u.id as user_id, u.global_canvas_id
+        user_info as (select p.unique_name,p.sis_user_id, '' as name, u.id as user_id, u.global_canvas_id
                     from (SELECT ROW_NUMBER() OVER (PARTITION BY user_id order by sis_user_id asc) AS row_number, * FROM pseudonym_dim) as p
                     join user_dim u on u.id = p.user_id WHERE row_number = 1),
         user_enroll as (select u.unique_name, u.sis_user_id, u.name, u.user_id, e.enroll_id,
                         u.global_canvas_id, e.type from enroll_data e join user_info u on e.user_id= u.user_id),
         course_fact as (select enrollment_id, current_score, final_score from course_score_fact
                         where course_id='{course_id}'),
-        final as (select cast(u.global_canvas_id as BIGINT) as user_id,u.name, u.unique_name as sis_name,
+        final as (select cast(u.global_canvas_id as BIGINT) as user_id,u.name, '' as sis_name,
```

Comment:
Can we omit the name stuff completely rather than using empty strings?

Reply:
Maybe they'd have to be nullable. Hm...

Reply:
Looks like the name and sis_name fields are indeed nullable. 

I have dropped those two fields from the cron queries.